ClosedXML.Report/Options/SummaryFuncTag.cs

```diff
@@ -6,10 +6,7 @@ namespace ClosedXML.Report.Options
 {
     public class SummaryFuncTag : OptionTag
     {
-        public override byte Priority
-        {
-            get { return 50; }
-        }
+        public override byte Priority => 50;
```

Comment:
Defining the `Priority` property on the tag level seems wrong to me. This way _each_ tag has to be aware of _all_ other tags in order to determine what priority it has to have. This includes not only an awareness of other tags priorities but also of their behavior and how they can affect each other. Far too tight coupling.

In my opinion, the priorities must be defined in `XLTemplate` where all tags are added to the registry. For implementor, this will be much easier this way to see which tags have which priorities, without a need to navigate between different classes.

Although, let's address this in a separate PR. For now, let it be like it is.